src/library/coverart.h

```diff
@@ -33,10 +33,13 @@ class CoverInfoRelative {
 
     // all-default memory management
     CoverInfoRelative(const CoverInfoRelative&) = default;
-    CoverInfoRelative(CoverInfoRelative&&) = default;
     CoverInfoRelative& operator=(const CoverInfoRelative&) = default;
-    CoverInfoRelative& operator=(CoverInfoRelative&&) = default;
     virtual ~CoverInfoRelative() = default;
+// Visual Studio does not support default generated move constructors
+#if !defined(_MSC_VER)
```

Comment:
This should include the version number since (hopefully) future versions of MSVC will work.